pkg/version/version.go

```diff
@@ -62,3 +65,18 @@ func GetVersion() Version {
 		Platform:  fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH),
 	}
 }
+
+// RegisterBuildInfo registers the build info metric.
+func RegisterBuildInfo(r prometheus.Registerer) {
```

Comment:
We can move this method to main.go instead. In `main.go`, we can call `version.GetVersion()`, get the version struct and use its fields to instantiate the Gauge vector in  `main.go`

Reply:
Yes, thanks! I will update it.